src/main/com/yetanalytics/squuid.cljc

```diff
@@ -1,4 +1,5 @@
 (ns com.yetanalytics.squuid
+  #?(:clj (:import [java.time Instant]))
   (:require [clojure.spec.alpha :as s]
             [com.yetanalytics.squuid.uuid :as u]
             [com.yetanalytics.squuid.time :as t]))
```

Comment:
Minor quibble but at Yet Analytics our convention is to place the `:import` _after_ the `:require` so I suggest you change that in order to be consistent with the rest of our codebase.